### PR TITLE
Remove whitespace from TODO message

### PR DESCRIPTION
**Pull request by wbagdon**

### CODE REVIEW

**Comment by jpsim** [comment]
could you please add a changelog entry?

File: Source/SwiftLintFramework/Rules/TodoRule.swift [hunk 1]
```diff
@@ -58,6 +58,8 @@ public struct TodoRule: ConfigurationProviderRule {
                 message = message.stringByReplacingOccurrencesOfString("TODO", withString: "")
             }
             message = message.stringByReplacingOccurrencesOfString("//", withString: "")
+            // trim whitespace
+            message = message.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceAndNewlineCharacterSet())
```
Comment (jpsim) [inline Source/SwiftLintFramework/Rules/TodoRule.swift]: please omit `NSCharacterSet`:

```swift
// trim whitespace
message = message.stringByTrimmingCharactersInSet(.whitespaceAndNewlineCharacterSet())
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,10 @@
 
 ##### Bug Fixes
 
+* Fixed whitespace being added to TODO messages
```
Comment (jpsim) [inline CHANGELOG.md]: missing period and two trailing spaces as defined in CONTRIBUTING.md

**Comment by wbagdon** [comment]
Would it make sense to remove the "TODOs should be avoided: " part of the message since this seems to be more of a feature now?

**Comment by jpsim** [comment]
Not sure about that, but PR looks good. Thanks @wbagdon!

**Pull request merged**